apps/src/lib/kits/maker/ui/EligibilityChecklist.jsx

```diff
@@ -110,6 +121,20 @@ export default class EligibilityChecklist extends Component {
             onSchoolConfirmed={this.handleSchoolConfirmed}
           />
         }
+        {this.state.discountAmount && !this.state.getsFullDiscount &&
+          <div style={styles.partialDiscountMessage}>
+            Acoording to our data, your school has fewer than 50% of students that are
+            eligible for free/reduced-price lunches. This means that we can bring down
+            the cost of the $325 kit to just &97.50.{" "}
+            <b>
+              If this data seems inaccurate and you believe there are over 50% of students
+              that are eligible for free/reduced-price lunch at your school, please contact
+              support@code.org.
+            </b>
+            {" "}
+            Otherwise, click "Get Code" below.
+          </div>
+        }
```

Comment:
Not strictly related to the rest of this PR, but something Poorva saw was missing when I walked her through the current state.

Chose to have the string here directly instead of in i18n, since (a) this page is for US only and (b) with various desired formatting it was easier not to bother with i18n.